resources/cluster-essentials/charts/pod-preset/values.yaml

```diff
@@ -0,0 +1,19 @@
+webhook:
+  replicaCount: 1
+  image:
+    repository: mszostok/service-catalog-admission-webhook
```

Comment:
additionally, it would be great to have README file. In such README I would like to know how to build new docker image.
Version 0.0.1 is build from which revision?

Reply:
done